src/components v2/index.ts

```diff
@@ -19,6 +21,8 @@ const components: PhotonComponents = {
   P2Message,
   P2InputCheckbox,
   P2InputCheckboxGroup,
+  P2RadioButton,
+  P2RadioButtonGroup,
```

Comment:
**issue:** We need the Input version of this as well.

Check the `P2InputCheckboxGroup` for more information.

Let's also do a quick 15 min session to go over this tomorrow.